graylog2-web-interface/src/views/components/sidebar/SidebarNavigation.tsx

```diff
@@ -83,14 +83,18 @@ const SidebarNavigation = ({ sections, activeSection, selectSidebarSection, side
                sidebarIsPinned={sidebarIsPinned} />
       <HorizontalRuleWrapper><hr /></HorizontalRuleWrapper>
       <SectionList>
-        {sections.map(({ key, icon, title }) => (
-          <NavItem isSelected={activeSectionKey === key}
-                   icon={icon}
-                   onClick={() => selectSidebarSection(key)}
-                   key={key}
-                   title={title}
-                   sidebarIsPinned={sidebarIsPinned} />
-        ))}
+        {sections.map(({ key, icon, title, CustomNavComponent }) => {
+          const NavComponent = CustomNavComponent || NavItem;
+
+          return (
+            <NavComponent isSelected={activeSectionKey === key}
+                          icon={icon}
+                          onClick={() => selectSidebarSection(key)}
+                          key={key}
+                          title={title}
+                          sidebarIsPinned={sidebarIsPinned} />
+          );
+        })}
```

Comment:
It would be nice to visually distinguish the these actions from the actual sidebar sections.

We could add a separator.
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/5ae47d05-984e-496f-8053-5d18ad3eb882)

Reply:
Beside the `sections` array, we could also have an `actions` array.

Reply:
We talked about removing the arrow in the sidebar, because you can open and close the sidebar by clicking in the section icon, but we can implement this in a different PR.